store/src/commonMain/kotlin/com/dropbox/external/store5/Market.kt

```diff
@@ -0,0 +1,24 @@
+package com.dropbox.external.store5
+
+import com.dropbox.external.store5.impl.RealMarket
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * Integrates stores and a bookkeeper.
+ * @see [RealMarket]
+ * @see [Store]
+ * @see [Bookkeeper]
+ */
+interface Market<Key : Any> {
```

Comment:
Does a user ever make their own market? If not maybe we can make internal

Reply:
Yes user will build their own `Market`. It will look like this https://github.com/matt-ramotar/Store/blob/main/samples/notes/android/src/main/kotlin/com/dropbox/market/notes/android/Graph.kt#L160-L165